client/client.go

```diff
@@ -94,14 +94,10 @@ type (
 		// NOTE: DO NOT USE THIS API INSIDE A WORKFLOW, USE workflow.ExecuteChildWorkflow instead
 		ExecuteWorkflow(ctx context.Context, options StartWorkflowOptions, workflow interface{}, args ...interface{}) (WorkflowRun, error)
 
-		// GetWorkfow retrieves a workflow execution and return a WorkflowRun instance (described above) and error
+		// GetWorkfow retrieves a workflow execution and return a WorkflowRun instance (described above)
```

Comment:
I would provide the full description even if it duplicates some of the data in the Execute document.